resolwe_bio/tests/processes/test_import.py

```diff
@@ -10,11 +10,10 @@ class ImportProcessorTestCase(BioProcessTestCase):
     @tag_process("import-sra", "import-sra-single", "import-sra-paired")
     def external_test_sra(self):
         # single-end reads from Polyak RNA-seq demo dataset
+        # prefetch needs to be disabled in tests to avoid downloading the whole SRA file bundle
```

Comment:
Would choosing a smaller file help? Perhaps a baby task for someone new to processes to get the feel of the workflow we have for working with processes/tests.

Reply:
It would, but one would need to find a really small sample (a few MB at most) in SRA for that.